src/Analyzer.Utilities/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Analyzer.Utilities.Extensions
+{
+    public static class StringExtensions
+    {
+        public static bool IsPlural(this string word)
+        {
+            if (!word.EndsWith("s", StringComparison.Ordinal) &&
```

Comment:
is this enough? what happen if code is like "DAYS" rather than "Days". 

by the way, if word finish with "i" or "ae", we consider it as plural?